ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/structure/footer/footer.html

```diff
@@ -0,0 +1,21 @@
+<footer class="cmp-footer">
+  <div class="cmp-footer__wrapper">
+    <div
+      class="cmp-footer__head"
+      data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
+    >
+      <sly
+        data-sly-resource="${'image' @ resourceType='/apps/aem-bootcamp/components/general/image', decoration=true, cssClassName='cmp-footer__logo-img'}"
```

Comment:
why are you adding the cssClassName property?

Reply:
needed because styles related to the footer design (_footer.scss)